Cubical/Algebra/DirectSum/DirectSumHIT/Base.agda

```diff
@@ -58,7 +58,7 @@ module _ (Idx : Type ℓ) (P : Idx → Type ℓ') (AGP : (r : Idx) → AbGroupSt
                      → PathP (λ i → Q (base-add r a b i)) ((base* r a) add* (base* r b)) (base* r (AbGroupStr._+_ (AGP r) a b)))
     where
 
-    f : (x : ⊕ Idx P AGP) → Q x
```

Comment:
'f' is the only thing defined in this module - don't you want to make the module anonymous and use its name instead of 'f'?

Reply:
will be fixed by a specific PR #822